test/unit/service/FileServiceSpec.scala

```diff
@@ -93,4 +93,37 @@ class FileServiceSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach
       await(service.publish(filePublishing)) shouldBe UnpublishedFileAttachment("id", "filename-updated", "type", 0, "Quarantined")
     }
   }
+
+  "Publish multiple" should {
```

Comment:
I would change the tests.
I would have the first item of the sequence failing and the second working (rather than the other way around - what you wrote now)

Reply:
Done